src/reducers/scoreboard.js

```diff
@@ -1,19 +1,30 @@
 import { scoresOrderOptions } from '../constants';
 
 const initialState = {
+  loading: false,
   pastScores: [],
+  error: null,
   numberOfCardsFilter: 'All',
-  scoresOrder: scoresOrderOptions.OLDEST_TO_NEWEST
+  scoresOrder: scoresOrderOptions.OLDEST_TO_NEWEST,
+  selectedChunkOfScores: 0
 }
 
 export const scoreboard = (state = initialState, action) => {
   switch(action.type) {
     case 'SET_PAST_SCORES':
       return { ...state, pastScores: action.payload };
+    case 'FETCH_PAST_SCORES_BEGIN':
```

Comment:
To jest spoko, jako własna implementacja ;) :+1: kiedyś dodamy libkę a teraz zmień tylko nazwę na ```FETCH_PAST_SCORES_PENDING```

Reply:
poprawione